spec/xcpretty/parser_spec.rb

```diff
@@ -180,6 +180,13 @@ module XCPretty
       @parser.parse(SAMPLE_PENDING_KIWI_TEST)
     end
 
+    it "parses measuring tests" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.